File: VB/DragDropExample/Gallery/CustomGalleryControlGalleryPainter.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.XtraBars.Ribbon.Drawing

Namespace DragDropExample.Gallery
	Public Class CustomGalleryControlGalleryPainter
		Inherits GalleryControlGalleryPainter
		Protected Overrides Function CreateGroupPainter() As GalleryItemGroupPainter
			Return New CustomGalleryItemGroupPainter()
		End Function
	End Class
End Namespace

File: CS/DragDropExample/Gallery/CustomGalleryControlGalleryPainter.cs
using DevExpress.XtraBars.Ribbon.Drawing;

namespace DragDropExample.Gallery {
    public class CustomGalleryControlGalleryPainter :GalleryControlGalleryPainter {
        protected override GalleryItemGroupPainter CreateGroupPainter() {
            return new CustomGalleryItemGroupPainter();
        }
    }
}
